cmd/describe/admin/cmd.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package admin
+
+import (
+	"os"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/moactl/pkg/aws"
+	"github.com/openshift/moactl/pkg/logging"
+	"github.com/openshift/moactl/pkg/ocm"
+	rprtr "github.com/openshift/moactl/pkg/reporter"
+)
+
+const (
+	idpName = "htpasswd"
+	// username = "cluster-admin"
+)
+
+var args struct {
+	clusterKey string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "admin [ID|NAME]",
+	Short: "Show details of the cluster-admin user",
+	Long:  "Show details of the cluster-admin user and a command to login to the cluster",
+	Example: `  # Describe cluster-admin user of a cluster names mycluster
+  moactl describe admin -c mycluster`,
+	Run: run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	flags.StringVarP(
+		&args.clusterKey,
+		"cluster",
+		"c",
+		"",
+		"Name or ID of the cluster to add the IdP to (required).",
```

Comment:
Name or ID of the cluster that cluster-admin belongs to

Reply:
done